src/config.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const { resolve } = require('path')
+
+/* c8 ignore next */
+require('dotenv').config({ path: process.env.ENV_FILE_PATH || resolve(process.cwd(), '.env') })
+
+const {
+  SNS_EVENTS_TOPIC: eventsTopic
```

Comment:
Please can you document this new env var on the README and mention that it needs to be the full topic ARN.

Reply:
done https://github.com/elastic-ipfs/bucket-to-indexer/pull/13/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R16